documentation/docs/25-build-and-deploy/40-adapter-node.md

```diff
@@ -116,7 +116,7 @@ We instead read from the _right_, accounting for the number of trusted proxies.
 
 ### `BODY_SIZE_LIMIT`
 
-The maximum request body size to accept in bytes including while streaming. Defaults to 512kb. You can disable this option with a value of `Infinity` (0 in older versions of the adapter) and implement a custom check in [`handle`](hooks#server-hooks-handle) if you need something more advanced.
+The maximum request body size to accept in bytes including while streaming. Defaults to 512kb. You can disable this option with a value of `Infinity` (0 in older versions of the adapter), set a custom value (numeric, in bytes) or implement a custom check in [`handle`](hooks#server-hooks-handle) if you need something more advanced.
```

Comment:
this is more wordy and the extra parens and comma make it a bit messy to read. how about just adding the numeric value?
```suggestion
The maximum request body size to accept in bytes including while streaming. Defaults to `524288` (512 kb). You can disable this option with a value of `Infinity` (0 in older versions of the adapter) and implement a custom check in [`handle`](hooks#server-hooks-handle) if you need something more advanced.
```